integration.go

```diff
@@ -133,6 +162,21 @@ func (client *Client) UpdateIntegrationAWS(identifier string, input AWSIntegrati
 	return m.Payload.Integration, HandleErrors(err, m.Payload.Errors)
 }
 
+func (client *Client) UpdateIntegrationNewRelic(resource IdentifierInput, input NewRelicIntegrationInput) (*Integration, error) {
```

Comment:
```suggestion
func (client *Client) UpdateIntegrationNewRelic(resource string, input NewRelicIntegrationInput) (*Integration, error) {
```

Reply:
Please use a string so that an ID or an Alias can be passed in.  This is more end user friendly.